integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -23,6 +24,8 @@ COPY --from=mbedos /opt/mbed-os /opt/mbed-os
 COPY --from=mbedos /opt/mbed-os-toolchain/ /opt/mbed-os-toolchain/
 COPY --from=mbedos /opt/openocd/ /opt/openocd/
 
+COPY --from=tizen /opt/tizen_sdk /opt/tizen_sdk
```

Comment:
since this updates the vscode image to be different from the previous version, please also bump up the chip-build/version  value.

Reply:
Version up to 0.4.34.

Reply:
@andy31415 We applied your review comment and the suggested version at `chip-build` is now 0.4.35. See the included update at `chip-build/version` below. 😄